test/uk/gov/hmrc/merchandiseinbaggage/controllers/GoodsOverThresholdControllerSpec.scala

```diff
@@ -33,6 +33,7 @@ class GoodsOverThresholdControllerSpec extends DeclarationJourneyControllerSpec
   private val view              = app.injector.instanceOf[GoodsOverThresholdView]
   private val calculatorService = app.injector.instanceOf[MibService]
   private val mibConnector      = app.injector.instanceOf[MibConnector]
+  private val stub              = app.injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these changes, thanks